src/components/feedback/index.tsx

```diff
@@ -103,6 +112,54 @@ export const Feedback = ({mode, tripPattern, quayListData}: FeedbackProps) => {
   const [selectedAlternativeIds, setSelectedAlternativeIds] = useState<
     number[]
   >([]);
+  const [displayStats, setDisplayStats] = useState<versionStats[]>([]);
+
+  const displayCountType = mode === 'departures' ? 'whitelist' : 'always';
```

Comment:
Synes dette blir for tett knyttet opp mot Avganger/`departures`. Jeg tror det er bedre om `Feedback`-komponenten ikke har slike knytninger. Kan dette flagget heller komme inn som Input-parameter?

Reply:
Har ordnet slik at allowList i seg selv er en optional prop på komponenten som i tillegg avgjør visningstype. Dokumenterte også i typen hva flagget gjør, med eksempel :)